Sources/Layouts/TextViewLayout.swift

```diff
@@ -130,6 +130,9 @@ open class TextViewLayout<TextView: UITextView>: BaseLayout<TextView>, Configura
     // MARK: - overriden methods
 
     open override func configure(view textView: TextView) {
+        // `isSelectable` is default to false, but it allows overriding in `configure` block
+        textView.isSelectable = false
```

Comment:
My point was that we should document why we are defaulting it to false (presumably because it cause some issue in iOS 8/9). If there is no longer a reason then we should remove this line entirely.

Reply:
After I retested iOS 8/9 and I couldn't find the issue anymore, so I will remove this entire line for the time being.